docs/getting-started/features.md

```diff
@@ -100,7 +98,19 @@ To perform a factory reset:
 1. Click **Factor Reset** to proceed. Kubernetes stops and Rancher Desktop closes.
 1. Optional: start Rancher Desktop again.
 
-# Preferences
+## Diagnostics
+
+**Note:** Rancher Desktop *doesn't* send the diagnostics data to any remote server for processing or storing.
+
+The **Diagnostics** feature runs several checks in the background to detect common problems such as missing minimum requirements, misconfigurations, etc., in your environment to help you self-troubleshoot and fix Rancher Desktop application issues.
+
+The diagnostics checks are run every time during the application launch. If there are any problems identified then the problems count is shown next to the *Diagnostics* menu text in the left navigation bar, indicating that something needs your attention. The *Diagnostics* tab displays the results of the diagnostics tests highlighting areas that need attention and guides you to resolve problems.
```

Comment:
I would add some commas, but maybe check with a docs person about that. 😄 

```suggestion
The diagnostics checks are run every time during the application launches. If there are any problems identified then the count of failed checks is shown next to the *Diagnostics* menu text in the left navigation bar, indicating that something needs your attention. The *Diagnostics* tab displays the results of the diagnostics tests, highlighting areas that need attention, and guides you to resolve problems.
```

Reply:
The commas do help in reading, especially with the long sentences that I tend to use.. :)